src/store/Transaction.ts

```diff
@@ -44,6 +45,54 @@ export enum TransactionGroupState {
     all = 'all',
 }
 
+export enum TransactionFilterOptions {
+    confirmed = 'isConfirmedSelected',
```

Comment:
just my own opinion.  would using number value + bitwise be better here? So you can skip below the long if / else etc...

https://www.w3schools.com/js/js_bitwise.asp

Reply:
I am using the property value of TransactionFilterOptions as key in TransactionFilterOptionsState